src/buildMathML.js

```diff
@@ -341,6 +341,31 @@ groupTypes.op = function(group, options) {
     return node;
 };
 
+groupTypes.mod = function(group, options) {
+    var inner = [];
+
+    if (group.value.modType === "pod" || group.value.modType === "pmod") {
+        inner.push(new mathMLTree.MathNode(
+            "mo", [makeText("(", group.mode)]));
+    }
+    if (group.value.modType !== "pod") {
+        inner.push(new mathMLTree.MathNode(
+            "mo", [makeText("mod", group.mode)]));
```

Comment:
If it's important that mathml have the same space widths as html I suppose I should repeat the logic from buildhtml here. I don't know, though, what the mathml is really for.